tests/VolleyManagement.UnitTests/Services/GameService/Fixture/GameServiceTestFixture.cs

```diff
@@ -913,6 +913,54 @@ public GameServiceTestFixture TestMinimumOddTeamsPlayOffSchedule()
             return this;
         }
 
+        public GameServiceTestFixture TestTeamsEqualsPlayOffScheme()
```

Comment:
TeamsEqualsPlayoffScheme is hard to understand

Reply:
What you think about SameTeamsInOneGamePlayOffScheme

Reply:
NoTeamsGameInPlayoff